testdata/scripts/deprecated-rewrite.txt

```diff
@@ -0,0 +1,13 @@
+cp foo.orig.go foo.go
```

Comment:
I'd say join the two txt files as `deprecated-flags.txt`, given how short and similar the tests are.